piccolo/columns/column_types.py

```diff
@@ -955,30 +956,33 @@ def __set__(self, obj, value: t.Union[datetime, None]):
 class Timestamptz(Column):
     """
     Used for storing timezone aware datetimes. Uses the ``datetime`` type for
-    values. The values are converted to UTC in the database, and are also
-    returned as UTC.
+    values. The values are converted to UTC when saved into the database and
+    are converted back into the timezone of the column on select queries.
 
     **Example**
 
     .. code-block:: python
 
         import datetime
+        from zoneinfo import ZoneInfo
 
-        class Concert(Table):
-            starts = Timestamptz()
+        class TallinnConcerts(Table):
+            event_start = Timestamptz(at_time_zone=ZoneInfo("Europe/Tallinn"))
 
         # Create
-        >>> await Concert(
-        ...    starts=datetime.datetime(
-        ...        year=2050, month=1, day=1, tzinfo=datetime.timezone.tz
```

Comment:
It seems that `tzinfo=datetime.timezone.tz` should be replaced with `tzinfo=datetime.timezone.utc` in the old documentation.